src/pymor/bindings/slycot.py

```diff
@@ -146,6 +162,39 @@ def solve_lyap_dense(A, E, B, trans=False, options=None):
     return X
 
 
+def solve_disc_lyap_dense(A, E, B, trans=False, options=None):
+    _solve_lyap_dense_check_args(A, E, B, trans)
+    options = _parse_options(options, lyap_dense_solver_options(), 'slycot_bartels-stewart', None, False)
+
+    if options['type'] == 'slycot_bartels-stewart':
+        n = A.shape[0]
+        C = -B.dot(B.T) if not trans else -B.T.dot(B)
+        trana = 'T' if not trans else 'N'
+        dico = 'D'
+        job = 'B'
+        if E is None:
+            # slycot does not set ldwork correctly for dico='D'
+            # hopefully this can be removed again
+            ldwork = 2*n*n+2*n
+            U = np.zeros((n, n))
+            X, scale, sep, ferr, _ = slycot.sb03md(n, C, A, U, dico, job=job, trana=trana, ldwork=ldwork)
```

Comment:
I have created a PR for this already https://github.com/python-control/Slycot/pull/166

Reply:
It has been merged into Slycot/master and should appear in the next release.